src/puppetlabs/ezbake/dependency_utils.clj

```diff
@@ -116,6 +117,21 @@
         (assoc acc project-name (.getInputStream jar-file jar-entry)))
       acc)))
 
+(defn get-stream-from-project-jar
+  [lein-project file-path]
+  (let [project-jar-file (fs/file (:target-path lein-project)
+                                  (str
+                                   (:name lein-project)
+                                   "-"
+                                   (:version lein-project)
+                                   ".jar"))]
+    (if (fs/exists? project-jar-file)
+      (let [project-jar (JarFile. project-jar-file)]
+        (if-let [project-jar-entry (find-file-in-jar project-jar
+                                                     file-path)]
+          (.getInputStream project-jar project-jar-entry)))
+      (log/warnf "Unable to find project jar file: '%s'" project-jar-file))))
```

Comment:
Sorry, not very familiar with this code.  Does this mean that it will always run when staging?  Would it then always run when building?  Also, I think the way this is written right now, the code would be looking for the local project jar and not the uberjar.  Do you think that's the right way to go - or no?